### PR TITLE
Components

### PR DESCRIPTION
**Pull request by yuvenalmash**
## changes summary
- remove boilerplate code
- create a calculator component
- Add styling to the calculator

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import './calculator.css';
+
+const Calculator = () => (
```
Comment (rloterh) [inline src/components/Calculator.js]: - [x] Please be reminded that this activity requires that your _Calculator_ component be a **stateful/ class-based** component. Kindly ensure to improve upon this aspect.

File: src/App.js [hunk 2]
```diff
@@ -1,29 +0,0 @@
-import logo from './logo.svg';
```
Comment (rloterh) [inline src/App.js:1]: The main component, `App.js`, is missing! Kindly ensure to add the default the _App_ component, which:
- [x] should be class-based.
- [x] should have the imported `Calculator.js` as seen in the code example below:

```javascript
// src/App.js

import React, { Component } from 'react';
import Calculator from ./components; // Import Calculator component

class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculator />
    };
  }
}
```

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Yuvenal, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦  🏃  

## Highlights 👍  
- Linter setup correctly!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- Pull request description should be in the past tense.
- Kindly ensure that the _Future Features_ section of your ReadMe includes 1-3 potential features to be added later.
- Demo link in your ReadMe is broken, kindly endeavor to fix it.
- Kindly endeavor to fill the FAQ section with at least 2 Q&A(s), or optionally remove this section.

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @yuvenalmash 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -53,22 +53,13 @@
...
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[Calculator]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (rloterh) [inline README.md:65]: - [ ] OPTIONAL
Live demo link on _line 72_ of your ReadMe is broken. Kindly endeavor to fix it!

**Review by rloterh — COMMENTED** [review]
## Status: Approved :heavy_check_mark:

Hi Abel,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Correctly implemented stateful/ class-based components!
- Descriptive Pull Request
- Good ReadMe

## Optional suggestions 🧰 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**